### PR TITLE
Yarn pnpm compatibility

### PR DESCRIPTION
**Pull request by rocambille**

### CODE REVIEW

**Comment by ydainna** [comment]
Hello it's posisble to verify and pass this PR please ? i verry interessted for this

File: package.json [hunk 1]
```diff
@@ -2,14 +2,17 @@
   "name": "template-fullstack",
   "version": "1.0.0",
   "description": "project template for the React-Express-MySQL cursus in Wild Code School",
+  "config": {
+    "cli": "npm"
+  },
```
Comment (LoicBrassart) [inline package.json:7]: (Heh, I didn't know of this config possibility)

**Review by LoicBrassart — APPROVED** [review]
Saw no Evil

File: package.json (hunk 1 above)
Comment (ydainna) [reply package.json:7]: I don't either lol

Comment (rocambille) [reply package.json:7]: neither do I 3 days ago : https://docs.npmjs.com/cli/v9/configuring-npm/package-json#config

**Comment by rocambille** [comment]
> Hello it's posisble to verify and pass this PR please ? i verry interessted for this

merged in dev branch with others things ;)

**Pull request merged**